npm-shrinkwrap.json

```diff
@@ -9211,6 +9211,14 @@
           "version": "1.1.0"
         }
       }
+    },
+    "source-map": {
```

Comment:
Yeah, this was generated with the `clingwrap` tool, whereas previously was probably part of `npm shrinkwrap --dev`. 

Per #3915, I think it's safe to omit the `resolved` and `from` fields, so this should not be a problem (cc @gwwar )